time time_stamp
float64 line_pos_f
float64 line_pos_b
float64[8] front_sensors
float64[8] back_sensors